R/recipe_run.R

```diff
@@ -514,30 +538,45 @@ orderly_prepare_data <- function(config, info, parameters, envir) {
   ## Compute the device stack size before starting work too
   n_dev <- length(grDevices::dev.list())
 
-  hash_resources <- hash_files(expand_directory_list(info$resources))
-  if (length(hash_resources) > 0L) {
-    orderly_log("resources",
-                sprintf("%s: %s", names(hash_resources), hash_resources))
-  } else {
-    hash_resources <- NULL
-  }
+  # hash_resources <- hash_files(expand_directory_list(info$resources))
+  # if (length(hash_resources) > 0L) {
+  #   orderly_log("resources",
+  #               sprintf("%s: %s", names(hash_resources), hash_resources))
+  # } else {
+  #   hash_resources <- NULL
+  # }
   missing_packages <- setdiff(info$packages, .packages(TRUE))
 
-  hash_global <- hash_files(expand_directory_list(info$global_resources))
-  if (length(hash_global) > 0L) {
-    orderly_log("global",
-                sprintf("%s: %s", names(hash_global), hash_global))
-  } else {
-    hash_global <- NULL
-  }
+  # hash_global <- hash_files(expand_directory_list(info$global_resources))
+  # if (length(hash_global) > 0L) {
+  #   orderly_log("global",
+  #               sprintf("%s: %s", names(hash_global), hash_global))
+  # } else {
+  #   hash_global <- NULL
+  # }
+  # 
+  # if (length(info$resources) > 0) {
+  #   size_resource <- file_size(expand_directory_list(info$resources))
+  # } else {
+  #   size_resource <- NULL
+  # }
+  # 
+  # if (length(info$global_resources) > 0) {
+  #   size_global <- file_size(expand_directory_list(info$global_resources))
+  # } else {
+  #   size_global <- NULL
+  # }
 
   if (length(missing_packages) > 0) {
     stop(paste("Missing packages:", 
                paste(squote(missing_packages), collapse = ", ")))
   }
 
-  ret <- list(data = ldata, hash_resources = hash_resources,
-              hash_global = hash_global, n_dev = n_dev)
+  # ret <- list(data = ldata, hash_resources = hash_resources,
+  #             hash_global = hash_global, n_dev = n_dev,
+  #             size_resource = size_resource, size_global = size_global)
+  
+  ret <- list(data = ldata, n_dev = n_dev)
```

Comment:
we no longer return hash_resources and hash_global here? were they not being used?